src/containerapp/azext_containerapp/_validators.py

```diff
@@ -216,11 +216,11 @@ def validate_target_port_range(cmd, namespace):
             raise ValidationError("Port must be in range [1, 65535].")
 
 
-def validate_timeout_in_seconds(cmd, namespace):
+def validate_session_timeout_in_seconds(cmd, namespace):
     timeout_in_seconds = namespace.timeout_in_seconds
     if timeout_in_seconds is not None:
-        if timeout_in_seconds < 0 or timeout_in_seconds > 180:
-            raise ValidationError("timeout in seconds must be in range [0, 180].")
+        if timeout_in_seconds <= 0 or timeout_in_seconds > 220:
+            raise ValidationError("Timeout in seconds must be in range [1, 220].")

```

Comment:
Do we have the same validation in RP?
If so, the customer can get the validate error from RP, and the CLI don't need to take this logic.

Reply:
I'd prefer leave it here since CLI anyway need update when run "az xxxx -h" for descriptions. RP has validation already. Let's consider remove if it is being frequently changed. I don't think it will further change based on the 220 value.